src/communication.ts

```diff
@@ -3,10 +3,12 @@ import { createSdkError, SdkErrors } from './services/errors';
 import { getSettings } from './settings';
 import { Message } from './util/message';
 
-export type Subscriber = (message: string) => void;
+export type Subscriber = (message: Message) => void;
 
 export class Communication {
     socketServer: Socket;
+    private static identifier: 0;
```

Comment:
make the identifier static and it will resolve your concern. @Rebal67

Reply:
fine by me